src/main/java/org/jboss/modules/JarFileResourceLoader.java

```diff
@@ -131,52 +131,52 @@ public synchronized ClassSpec getClassSpec(final String fileName) throws IOExcep
             // no such entry
             return null;
         }
-            final ClassSpec spec = new ClassSpec();
```

Comment:
Doing this as a separate step is not necessary since we can always do a diff with the `-w` flag, or (in GitHub) add `?w=1` to the query parameters, to view the diff without whitespace changes.